.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+script: 
+    - "! rm -rf !(telethon|telethon_generator|LICENSE|MANIFEST.in|README.rst|setup.py) .*"
```

Comment:
Not sure I understand why it removes everything?

Reply:
It removes all, except telethon, telethon_generator, LICENSE, MANIFEST.in, README.rst and setup.py. 

This command cleans your repo to push it to pypi.

Now I think the better is
```
"rm -rf !(telethon|telethon_generator|LICENSE|MANIFEST.in|README.rst|setup.py) .* | true"
```